sycl/test/CMakeLists.txt

```diff
@@ -1,23 +1,11 @@
-set(LLVM_BUILD_LIBRARY_DIRS "${LLVM_BINARY_DIR}/lib/")
-set(LLVM_BUILD_BINARY_DIRS "${LLVM_BINARY_DIR}/bin/")
 set(LLVM_TOOLS_DIR "${LLVM_BINARY_DIR}/bin/")
-set(CLANG_IN_BUILD "${LLVM_BINARY_DIR}/bin/clang")
-set(CLANGXX_IN_BUILD "${LLVM_BINARY_DIR}/bin/clang++")
-set(CLANGCL_IN_BUILD "${LLVM_BINARY_DIR}/bin/clang-cl")
-
-set(LLVM_DEPLOY_LIBRARY_DIRS "${CMAKE_INSTALL_PREFIX}/lib/")
-set(LLVM_DEPLOY_BINARY_DIRS "${CMAKE_INSTALL_PREFIX}/bin/")
-set(CLANG_IN_DEPLOY "${CMAKE_INSTALL_PREFIX}/bin/clang")
-set(CLANGXX_IN_DEPLOY "${CMAKE_INSTALL_PREFIX}/bin/clang++")
-set(CLANGCL_IN_DEPLOY "${CMAKE_INSTALL_PREFIX}/bin/clang-cl")
 
 get_target_property(SYCL_BINARY_DIR sycl-toolchain BINARY_DIR)
 
 set(SYCL_INCLUDE "${dst_dir}")
-set(SYCL_DEPLOY_INCLUDE "${dst_deploy_dir}")
 
 set(RT_TEST_ARGS ${RT_TEST_ARGS} "-v")
-set(DEPLOY_RT_TEST_ARGS ${DEPLOY_RT_TEST_ARGS} "-v --config-prefix=deploy-lit")
+set(DEPLOY_RT_TEST_ARGS ${DEPLOY_RT_TEST_ARGS} "-v -D SYCL_TOOLS_DIR=${CMAKE_INSTALL_PREFIX}/bin -D SYCL_LIBS_DIR=${CMAKE_INSTALL_PREFIX}/lib -D SYCL_INCLUDE=${dst_deploy_dir}")
```

Comment:
```suggestion
set(DEPLOY_RT_TEST_ARGS ${DEPLOY_RT_TEST_ARGS} "-v -D SYCL_TOOLS_DIR=${CMAKE_INSTALL_PREFIX}/bin -D SYCL_LIBS_DIR=${CMAKE_INSTALL_PREFIX}/lib${LLVM_LIBDIR_SUFFIX} -D SYCL_INCLUDE=${dst_deploy_dir}")
```

Reply:
I'm going to commit it separately. Thanks!

Reply:
https://github.com/intel/llvm/pull/1228